genome_designer/genome_finish/millstone_de_novo_fns.py

```diff
@@ -4,6 +4,8 @@
 import pickle
 import re
 
+from Bio import SeqIO
+
 from settings import TOOLS_DIR
```

Comment:
Just noticed this. Instead do:

    from django.conf import settings

And then wherever you use these do

    settings.TOOLS_DIR

etc.